datasets/samplemotordatalimerick.py

```diff
@@ -0,0 +1,662 @@
+###################################################################################################
+#
+# Copyright (C) 2024 Analog Devices, Inc. All Rights Reserved.
+# This software is proprietary to Analog Devices, Inc. and its licensors.
+#
+###################################################################################################
+"""
+Classes and functions for the Limerick Sample Motor Data Dataset
+"""
+import errno
+import math
+import os
+import pickle
+
+import numpy as np
+import torch
+from numpy.fft import fft
+from torch.utils.data import Dataset
+from torchvision import transforms
+
+import pandas as pd
+import scipy
+
+import ai8x
+
+
+class SampleMotorDataLimerick(Dataset):
+    """
+    ADXL356C
+    """
+
+    # Order 0 is used for default sensor: ADX365C
+    sensor_options = ['ADXL356C']
+    # Order 0 is reserved for 'all' do not change order
+    rpm_options = ('all', '0600', '1200', '1800', '2400', '3000')
+    # Order 0 is used for default sensor: ADX365C, do not change order
+    sensor_options_sr_Hz = [20000]
```

Comment:
we can remove all options and keep sampling rate and data len as variables

Reply:
Sensor and rpm selection is removed